docs/guides/scaffolding.md

```diff
@@ -5,98 +5,77 @@ Developing with the {{open_autonomy}} framework usually entails creating all the
-- ROUND_TIMEOUT
-default_start_state: RegistrationRound
-final_states: []
-label: packages.valory.skills.hello_world_abci.rounds.HelloWorldAbciApp
-start_states:
-- RegistrationRound
-states:
-- PrintMessageRound
-- RegistrationRound
-- ResetAndPauseRound
-- SelectKeeperRound
-transition_func:
-    (PrintMessageRound, DONE): ResetAndPauseRound
-    (PrintMessageRound, ROUND_TIMEOUT): RegistrationRound
-    (RegistrationRound, DONE): SelectKeeperRound
-    (ResetAndPauseRound, DONE): SelectKeeperRound
-    (ResetAndPauseRound, NO_MAJORITY): RegistrationRound
-    (ResetAndPauseRound, RESET_TIMEOUT): RegistrationRound
-    (SelectKeeperRound, DONE): PrintMessageRound
-    (SelectKeeperRound, NO_MAJORITY): RegistrationRound
-    (SelectKeeperRound, ROUND_TIMEOUT): RegistrationRound
-```
-
-Now, run the scaffold tool:
+3. **Generate the template classes.** Using the scaffold tool to generate the skeleton for the classes:
+    ```bash
+    autonomy scaffold fsm my_skill --spec fsm_specification.yaml
+    ```
+    You will see that the generated rounds, payloads and behaviours already appear with their correct names, as well as the `HelloWorldAbciApp` and its transition function.
 
-```
-autonomy scaffold fsm my_skill --spec fsm_specification.yaml
-```
+4. **Push the generated skill to the local registry.** Now that you have successfully created the skill, you can push it to the local registry. Since to fetch all packages from IPFS we had used the remote registry, we now need to reset it so we use a local one during the development:
 
-You will see that the generated rounds, payloads and behaviours already appear with their correct names, as well as the `HelloWorldAbciApp` and its transition function. Now that you have successfully created the skill, you can push it to the local registry. Since to fetch all packages from IPFS we had used the remote registry, we now need to reset it so we use a local one during the development:
-```
-autonomy init --local --reset --author AUTHOR
-```
+    ```bash
+    autonomy init --local --reset --author AUTHOR
+    ```
 
-To use the local registry, it is expected that a `packages` directory exists at the same level as our agent. If you already have it then you can skip this step, but if you don't then you need to create it. In this example, we need to create `my_workspace/packages`.
-```
-cd ..
-mkdir packages
-cd my_agent
-```
+    To use the local registry, it is expected that a `packages` directory exists at the same level as our agent. If you already have it then you can skip this step, but if you don't then you need to create it. In this example, we need to create `my_workspace/packages`.
+    ```bash
+    cd ..
+    mkdir packages
+    cd my_agent
+    ```
 
-And we can now push our new skill to the local registry:
-```
-autonomy push skill <your_author_name>/my_skill
-```
+    And we can now push our new skill to the local registry:
+    ```bash
+    autonomy push skill <your_author_name>/my_skill
```

Comment:
```suggestion
    autonomy push skill <your_author_name>/my_skill --local
```
otherwise it will be in remote ... please confirm position of local flag is correct

Reply:
checked.